web/app.fluidity.money/app/styles/dashboard.css

```diff
@@ -461,9 +461,9 @@ main > footer > section > a {
 
 .provide-liquidity-left > div:first-child {
   display: flex;
-  flex-direction: column;
+  flex-direction: column-reverse;
   gap: 0.5em;
```

Comment:
I added this so when you scroll down on mobile/desktop, “FLUIDIFY MONEY” button does not cover any components. If this does not already happen, please readd